xrmocap/ops/triangulation/jacobi_triangulator.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+from typing import Union 
+from xrmocap.utils.triangulation_utils import prepare_triangulate_input
+
+class JacobiTriangulator():
+
+    def __init__(self,maxIterTime=20, updateTolerance=1e-4, regularTerm=1e-4, logger=None):
```

Comment:
Someone may construct a triangulator by
```python
tri = JacobiTriangulator(camera_parameters=camera_parameters)
```
instead of
```python
tri = JacobiTriangulator()
tri.set_cameras(camera_parameters)
```
Thus we need `camera_parameters` in `__init__()`